### PR TITLE
rpm: enable python

### PR DESCRIPTION
**Pull request by dawidd6**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Some users (like me) might want to be able to use `rpm` python library.

### CODE REVIEW

File: Formula/rpm.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Rpm < Formula
   homepage "https://rpm.org/"
   url "http://ftp.rpm.org/releases/rpm-4.15.x/rpm-4.15.1.tar.bz2"
   sha256 "ddef45f9601cd12042edfc9b6e37efcca32814e1e0f4bb8682d08144a3e2d230"
-  version_scheme 1
+  version_scheme 2
```
Comment (Bo98) [inline Formula/rpm.rb]: Shouldn't the revision be bumped instead?

Comment (dawidd6) [reply Formula/rpm.rb]: To be honest I don't know what `version_scheme` is for. Thought it's some fancy revision-like function.

Comment (Bo98) [reply Formula/rpm.rb]: In some sense it is. It's typically when upstream changes their versioning system and the new version might be seen by `brew` as an earlier version. For example, if a project switched from YYYY.MM.DD format to semver then we'll need to bump version_scheme. Basically the order of comparisons is `version_scheme`, `version`, `revision`.

`revision` is reset on version upgrades. `version_scheme` is never ever reset.

**Comment by dawidd6** [comment]
```
rpm:
  * python modules have explicit framework links
    These python extension modules were linked directly to a Python
    framework binary. They should be linked with -undefined dynamic_lookup
    instead of -lpython or -framework Python.
      /usr/local/opt/rpm/lib/python3.7/site-packages/rpm/_rpm.so
```

Dunno what does that mean, tried to add `-undefined dynamic_lookup` to `LDFLAGS`, but then another error popped out.

**Pull request closed without merging**